example/example_parser_cia.py

```diff
@@ -0,0 +1,10 @@
+import cdp
+
+class MyCDPParameter(cdp.cdp_parameter.CDPParameter):
+     def check_values(self):
+          pass
+
+
+parser = cdp.cdp_parser.CDPParser(MyCDPParameter(),["/export/gleckler1/git/CIA/Defaults/DefArgsCIA.json",])
```

Comment:
This hard coded (and machine dependent) path is a BAAAAAD idea

Reply:
Thanks, we'll fix this. @doutriaux1